test/models/Reads/schemes/EstablisherInfoReadsSpec.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.Reads.schemes
+
+import models.Samples
+import models.schemes.{CompanyDetails, EstablisherInfo, IndividualDetails, PartnershipDetails}
+import org.scalatest.{MustMatchers, OptionValues, WordSpec}
+
+class EstablisherInfoReadsSpec extends WordSpec with MustMatchers with OptionValues with Samples with SchemeDetailsStubJsonData {
+
+  "A JSON payload containing establisher details" should {
+
+    "read into a valid establisher details object" when {
+
+      val result = establisherDetails.as(EstablisherInfo.apiReads)
```

Comment:
I'd give this a different name as it's used in multiple tests below and makes understanding the tests slightly confusing.